File: TAbaretata/JavaScript-Fundamentals-2015/05.Arrays/01.AllocateArray/AllocateArray.js

// Problem 1: Write a script that allocates array of 20 integers and initializes each element by 
// its index multiplied by 5. Print the obtained array on the console.

var arr = new Array(20);

for (var i = 0; i < arr.length; i++)
{
    arr[i] = i * 5;
    console.log('arr[' + i + '] = ' + arr[i]);
}

File: TAbaretata/CSharpPartTwo-2015/Arrays/1.AllocateArray/Allocate array.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

//Write a program that allocates array of 20 integers and initializes each element by its index multiplied by 5.

class AllocateArray
{
    static void Main()
    {
        int[] arr = new int[20];
        for (int i = 0; i < arr.Length; i++)
        {
            arr[i] = i * 5;
            Console.WriteLine("arr[{0}] = {1}", i, arr[i]);
        }
    }
}